test/e2e_node/e2e_service.go

```diff
@@ -314,9 +314,15 @@ type killCmd struct {
 }
 
 func (k *killCmd) Kill() error {
+	if k == nil {
+		glog.V(2).Infof("The killCmd is nil, nothing will be killed")
```

Comment:
I guess there was discussion on this but IMO calling methods on nil pointers is at the very least a bad code smell, and it should be documented very clearly as explicitly allowed.